ansible/group_vars/all

```diff
@@ -5,4 +5,5 @@ histomicstk_version: master
 large_image_version: master
 dsa_version: master
 girder_worker_version: master
+itk_version: master
```

Comment:
Is it possible to auto-get the latest release like [here](http://julienrenaux.fr/2013/10/04/how-to-automatically-checkout-the-latest-tag-of-a-git-repository/) so we don't have to track/watch those repositories?

Should we get the master for all repos within DigitalSlideArchive (since they are closely dependent) and peg to latest release tags for external repositories (ex: ITK) only?